statistics/cmsketch.go

```diff
@@ -919,11 +920,11 @@ func checkEmptyTopNs(topNs []*TopN) bool {
 
 // SortTopnMeta sort topnMeta
 func SortTopnMeta(topnMetas []TopNMeta) []TopNMeta {
-	slices.SortFunc(topnMetas, func(i, j TopNMeta) bool {
+	slices.SortFunc(topnMetas, func(i, j TopNMeta) int {
 		if i.Count != j.Count {
-			return i.Count > j.Count
+			return cmp.Compare(i.Count, j.Count)
```

Comment:
I think this is the opposite of the original logic. Also, could you please add some tests to cover this?

Reply:
yes，need desc order